openstates/de/legislators.py

```diff
@@ -68,64 +92,64 @@ def scrape_bio(self, term, chamber, district, name, url):
         return leg
 
     def scrape_contact_info(self, doc):
-        office_names = ['Legislative Hall Office', 'Outside Office']
-        office_types = ['capitol', 'district']
-        xpath = '//u[contains(., "Office")]/ancestor::table/tr[2]/td'
-        data = zip(doc.xpath(xpath)[::2], office_names, office_types)
-        info = {}
 
         # Email
-        xpath = '//font[contains(., "E-mail Address")]/../font[2]'
-        email = doc.xpath(xpath)[0].text_content()
-
-        # If multiple email addresses listed, only take the official
-        # noone@state.de.us address.
-        emails = re.split(r'(?:\n| or |;|\s+)', email)
-        for email in filter(None, emails):
-            if email.strip():
-                info['email'] = email.strip()
-                break
+        email = doc.xpath(".//a[contains(@href,'mailto')]")
+        email = email[0].text_content().strip()
```

Comment:
Sometimes this `text_content()` will capture two email addresses, not just one. For example, [Sokola](http://legis.delaware.gov//legislature.nsf/vwMain/E5D26F90AC51457285257E0E005A56C6?Opendocument). This seems to be remedied if you take the `@href` instead.